### PR TITLE
Consensus ParaTimes dashboard section

### PR DESCRIPTION
**Pull request by buberdds**

### CODE REVIEW

File: src/app/pages/ConsensusDashboardPage/ParaTimesCard.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+  return (
+    <Card>
+      <CardHeader
+        disableTypography
+        component="h3"
+        title={
+          <CardHeaderWithCounter
+            label={t('paratimes.listTitle')}
+            totalCount={3}
```
Comment (lukaw3d) [inline src/app/pages/ConsensusDashboardPage/ParaTimesCard.tsx:52]: Should count enabled+disabled runtimes until we have API

File: src/app/pages/ConsensusDashboardPage/RuntimePreview.tsx [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+      <Box sx={{ display: 'flex', justifyContent: 'center', paddingTop: 3 }}>
+        {status && (
+          <>
+            <PanelButton
+              activePanel={panel}
+              ariaLabel="Transactions"
+              panel="transactions"
+              setPanel={setPanel}
+            />
+            <PanelButton activePanel={panel} ariaLabel="Accounts" panel="accounts" setPanel={setPanel} />
```
Comment (lukaw3d) [inline src/app/pages/ConsensusDashboardPage/RuntimePreview.tsx:184]: Translations

File: src/app/utils/route-utils.ts [hunk 3]
```diff
@@ -124,6 +124,20 @@ export abstract class RouteUtils {
     return Object.values(Layer).filter(layer => RouteUtils.ENABLED_LAYERS_FOR_NETWORK[network][layer])
   }
 
+  static getRuntimesForNetwork(network: Network, isEnabled: boolean): Runtime[] {
+    return Object.values(Runtime).filter(
+      runtime => RouteUtils.ENABLED_LAYERS_FOR_NETWORK[network][runtime] === isEnabled,
+    )
+  }
+
+  static getEnabledRuntimesForNetwork(network: Network): Runtime[] {
```
Comment (lukaw3d) [inline src/app/utils/route-utils.ts:133]: This seems too easily confused with getEnabledLayersForNetwork.

I would refactor all these into: `getAllLayersForNetwork(network): {enabled: [], disabled: []}` + ignore consensus where it is consumed.

File: src/locales/en/translation.json [hunk 4]
```diff
@@ -479,6 +479,15 @@
     "coming": "Coming",
     "soon": "soon"
   },
+  "paratimes": {
+    "active": "Active",
+    "activeNodes": "{{nodes}} active",
+    "blockNumber": "Block number:",
+    "inactive": "Suspended",
```
Comment (lukaw3d) [inline src/locales/en/translation.json:486]: @donouwens this implies that Cipher runtime is suspended, not just that indexer doesn't support it.
image

do we need more status options, or is this just to demo how suspended ones will look like, and we should change Cipher to "Active" but with no data?

Comment (buberdds) [reply src/locales/en/translation.json:505]: @donouwens, we don't want to show new ParaTime Pontus-X here, right?

File: src/app/utils/route-utils.ts (hunk 3 above)
Comment (buberdds) [reply src/app/utils/route-utils.ts]: done

File: src/locales/en/translation.json (hunk 4 above)
Comment (buberdds) [reply src/locales/en/translation.json]: I have changed "Suspended" to "Coming soon" for now. My preference would be to remove status row entirely.

Comment (lukaw3d) [reply src/locales/en/translation.json]: I expect in the future we are supposed to get data from API about the *actual* status of runtimes

File: src/app/pages/ConsensusDashboardPage/RuntimePreview.tsx [hunk 5]
```diff
@@ -0,0 +1,211 @@
...
+      </StyledList>
+      <StyledBox>
+        <Box gap={3} sx={{ flex: 1, display: 'flex', flexDirection: 'column', alignItems: 'center' }}>
+          {status && (
+            <Box sx={{ width: '100%' }}>
+              {panel === 'transactions' && (
+                <TransactionsChartCard
+                  scope={{
+                    layer: runtime,
+                    network: 'mainnet',
```
Comment (lukaw3d) [inline src/app/pages/ConsensusDashboardPage/RuntimePreview.tsx:146]: TODO: un-hardcode mainnet

**Review by lukaw3d — APPROVED** [review]

File: src/locales/en/translation.json (hunk 4 above)
Comment (buberdds) [reply src/locales/en/translation.json]: Oh I thought it's rather avail/or not avail in Explorer, but yeah useRuntimeFreshness makes sense too. Will add a ticket to address this in the future.

**Pull request merged**